src/framework/scene-registry.js

```diff
@@ -269,6 +269,7 @@ class SceneRegistry {
                 // initialize components
                 self._app.systems.fire('initialize', entity);
                 self._app.systems.fire('postInitialize', entity);
+                self._app.systems.fire('initializeEntityReferences', entity);
```

Comment:
Don't think you need this new event.

Just call `self._app.fire('postinitialize');`here instead of the new one. (Since that's what entity-reference is listening for).

Reply:
We could. It does feel weird firing an app level event every time a user loads a scene at runtime, especially when this wasn't the behaviour before PR #3557

I can't find anything in the engine listening to the app initialize and postinitialize events but feel it's much safer to have a separate event like it did before PR #3557